frontend/language/src/nb.json

```
@@ -636,20 +636,19 @@
   "preview.subheader.showas": "Vis som",
   "preview.view_size_desktop": "Desktop",
   "preview.view_size_mobile": "Mobil",
-  "process_editor.configuration_panel_confirmation_task": "Task: Bekreftelse",
-  "process_editor.configuration_panel_data_task": "Task: Utfylling",
-  "process_editor.configuration_panel_element_not_supported": "Vi støtter foreløpig kun å se på Tasks. Velg en Task for å se detaljer.",
-  "process_editor.configuration_panel_feedback_task": "Task: Tilbakemelding",
-  "process_editor.configuration_panel_header_help_text_confirmation": "Bekreftelse-task (confirmation) brukes for å vise sluttbruker en oppsummering som de kan bekrefte.",
-  "process_editor.configuration_panel_header_help_text_data": "Utfylling-task (data) brukes dersom det skal samles inn eller vises frem data.",
-  "process_editor.configuration_panel_header_help_text_feedback": "Tilbakemelding-task (feedback) brukes når sluttbruker må vente på tilbakemelding fra tjenesteeier før de kan gå videre i, eller avslutte, prosessen.",
-  "process_editor.configuration_panel_header_help_text_signing": "Signerings-task (signing) brukes når sluttbruker skal bekrefte med signatur.",
-  "process_editor.configuration_panel_header_help_text_title": "Informasjon om valgt task",
+  "process_editor.configuration_panel_confirmation_task": "Oppgave: Bekreftelse",
+  "process_editor.configuration_panel_data_task": "Oppgave: Utfylling",
+  "process_editor.configuration_panel_element_not_supported": "Vi støtter foreløpig kun å se på Oppgaver. Velg en Oppgave for å se detaljer.",
```

Suggest we use lowercase for "oppgave" in this text
```suggestion
  "process_editor.configuration_panel_element_not_supported": "Vi støtter foreløpig kun å se på oppgaver. Velg en oppgave for å se detaljer.",
```